Client/Extensions/UIAlertControllerExtensions.swift

```diff
@@ -10,6 +10,14 @@ typealias UIAlertActionCallback = (UIAlertAction) -> Void
 extension UIAlertController {
 
     /**
+     Allows the keyboard to pop back up after an alertview.
+     Might have adverse effects in other places. Need to check.
```

Comment:
I wouldn't put this here since this could effect other consumers of `UIAlertController`, like you said. Just override the property in the specific controllers that need it.